Sources/ProjectDescription/MergedBinaryType.swift

```diff
@@ -0,0 +1,34 @@
+/// Represents the different options to configure a target for mergeable libraries
+///
+/// https://developer.apple.com/documentation/xcode/configuring-your-project-to-use-mergeable-libraries
+public enum MergedBinaryType: Equatable, Codable {
+    /// Target is never going to merge available dependencies
+    case disabled
+
+    /// Target is going to merge direct target dependencies (just the ones declared as part of it's project). With this build setting,
+    /// Xcode treats mergeable dependencies like normal dynamic libraries in debug builds,
+    /// but performs steps in release mode to automatically handle merging for **direct dependencies**
+    ///
+    /// A direct dependency is a library that meets two criteria:
+    /// - The library is listed in your target’s Link Binary with Libraries build phase.
+    /// - The library is the product of another target in your project.
```

Comment:
ah interesting I missed this! am I correct in understanding we can't form merged libraries across projects in a workspace 🤐?

Reply:
I guess it will be able to link between xcodeproj via workspaces. Because what Apple says it is considering dependencies that are targets... but I've never tried it 😅